### PR TITLE
ref: use default GlowKit BoundingBox & nullability annotations

### PR DESCRIPTION
**Pull request by VaiTon**
1. Use the default GlowKit/Bukkit implementation of BoundingBox instead of using a custom one.
2. Added some more nullability annotations.

### CODE REVIEW

File: src/main/java/net/glowstone/GlowWorld.java [hunk 1]
```diff
@@ -15,24 +15,11 @@
...
 import net.glowstone.chunk.GlowChunk.Key;
 import net.glowstone.chunk.GlowChunkSnapshot.EmptySnapshot;
-import net.glowstone.constants.GameRules;
-import net.glowstone.constants.GlowBiome;
-import net.glowstone.constants.GlowBiomeClimate;
-import net.glowstone.constants.GlowEffect;
-import net.glowstone.constants.GlowParticle;
-import net.glowstone.constants.GlowSound;
-import net.glowstone.constants.GlowTree;
+import net.glowstone.constants.*;
```
Comment (mastercoms) [inline src/main/java/net/glowstone/GlowWorld.java]: No wildcard imports.

**Review by mastercoms — CHANGES_REQUESTED** [review]
Everything else looks good to me, though I'll have to do a second pass later

**Review by mastercoms — CHANGES_REQUESTED** [review]
Are the java. import ordering changes correct?

**Comment by VaiTon** [comment]
@mastercoms they should be as I'm using checkstyle. I'm merging the conflicts!

**Pull request closed without merging**